src/dowel/tensor_board_output.py

```diff
@@ -46,10 +45,12 @@ class TensorBoardOutput(LogOutput):
 
     def __init__(self,
                  log_dir,
+                 keys_accepted=r'^\S+$',
```

Comment:
is there a reason you didn't use `r'^$'` (which matches all strings)? I think TensorBoard will slugify any whitespace (not saying it's good practice, but I don't think it barfs if you do it).